### ISSUE TITLE
Embed video - in post - not responsive

### ISSUE THREAD

**Issue by VadymVolos** [Obox/layerswp#294]
Hi! :)
![112](https://cloud.githubusercontent.com/assets/12669290/11301273/87be46e0-8f9e-11e5-943b-7c4b0b30de84.png)
![113](https://cloud.githubusercontent.com/assets/12669290/11301274/87cc28c8-8f9e-11e5-9b69-512cf68071db.png)
![114](https://cloud.githubusercontent.com/assets/12669290/11301272/879faa8c-8f9e-11e5-9314-b3c2c2e589d3.png)

**Comment by VadymVolos** [comment]
I did (It works, but the error remained.):
```css
.blog .thumbnail.push-bottom iframe {
    display: block;
    overflow: hidden;
    z-index: 1;
    position: absolute;
    top: 0;
    left: 0;
    width: 100%;
    height: 100%;
}
.single .thumbnail.push-bottom iframe {
    height: 100%;
    min-height: 371px;
}
```

**Comment by vailjoy** [comment]
Hi there,

1. If I test by embedding the video using the Layers video embed field on the post, I do not get any console errors and the video loads as expected. Note that the size is fluid, so you may get black bars on the video player depending on your sidebar configuration and screen size.

2. If I test by simply dropping the video share url into the post content, WordPress oEmbed takes over and displays the video with no console errors and constrains the player to the aspect of the video.

These are the two ways to embed video into a post in Layers - one uses a generalized function to replace the WordPress featured image with your video embed, the other uses WordPress oembed to display it wherever you place the URL in your post content. Neither use jQuery in Layers. If you continue having issues with the current version of Layers (1.2.9) please try testing with the above steps on an install with no active plugins or child themes to eliminate them as the problem cause.

**Comment by VadymVolos** [comment]
https://youtu.be/Yr3tv1hW1gg

![113](https://cloud.githubusercontent.com/assets/12669290/11590993/17829e08-9a9d-11e5-925a-7aa15ee938d0.jpg)

![114](https://cloud.githubusercontent.com/assets/12669290/11590999/1c80ee8c-9a9d-11e5-8696-f881619b4f4c.jpg)

**Comment by vailjoy** [comment]
Fitvids has been implemented for video in posts in 1.2.13
